src/shared/log-messages.ts

```diff
@@ -27,6 +27,11 @@ export enum LogWarnings {
   signingEventSignatureNotRecognized = "It is not possible to add a signature to the event.",
   collectMetricsSignatureNotRecognized = "It is not possible to collect metrics from this event.",
   documentationSkeletonConfigNotFound = "No documentation skeleton config file found. Owner/Repo is: '%s'",
+  documentationSkeletonConfigNotParsable = "Could not parse content of documentation skeleton config for owner/repo: '%s'",
+  documentationSkeletonConfigNonStringVersion = "Focumentation skeleton config version field does not contain string. Owner/Repo is: '%s'",
```

Comment:
Typo
"Focumentation" -> "Documentation"
FYI I Use the [code spell checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) VS Code extension to catch stuff like that. Not sure though it if can differentiate between BE and AE.

Reply:
What do you mean, that's what I call the documentation skeleton, it's a regional dialect from upstate New York…

Thanks for finding this :D I tried using the extension last year but couldn't deal so well with the constant noise from all the false positives.